rts/src/main/java/eta/runtime/RuntimeOptions.java

```diff
@@ -0,0 +1,74 @@
+package eta.runtime;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class RuntimeOptions {
+    private Properties p;
+
+    public RuntimeOptions(String path) {
+        load_properties(path);
+    }
+
+    /*
+     * Loads properties from resource which
+     * may be override by system properties
+     */
+    private void load_properties(String path) {
```

Comment:
Please name this as `loadProperties` according to the naming convention of the rest of the source code.